.vscode/launch.json

```diff
@@ -1,11 +1,35 @@
+// A launch configuration that compiles the extension and then opens it inside a new window
+// Use IntelliSense to learn about possible attributes.
+// Hover to view descriptions of existing attributes.
+// For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
I think comments are technically invalid in json so we might want to remove these

Reply:
This file, at least originally, is vscode generated, so I'm inclined to say, if they don't mind comments, I don't either.